### PR TITLE
Add `include_in_every_saturn` 

### PR DESCRIPTION
**Pull request by jsignell**
Based on https://github.com/saturncloud/decision-records/pull/33, we are going to be adding this field to Saturn proper. I think this can be done in isolation, but I'll let integration tests prove me right or wrong :)

### CODE REVIEW

**Comment by jsignell** [comment]
Integration tests run: https://github.com/saturncloud/release-images/actions/runs/770970573

**Review by jameslamb — CHANGES_REQUESTED** [review]
Could you please update https://github.com/saturncloud/examples/blob/main/CONTRIBUTING.md#saturnsaturnjson- with details on what this field does?

I'd also welcome a separate PR to update that with the details of other fields that have been added since that was first written (`title`, `thumbnail_image_url`, `weight`).

**Comment by jsignell** [comment]
Ooh yeah, good thinking.

**Comment by jsignell** [comment]
The integration tests ran fine, they didn't quite pass, but that's due to flakiness not this change.

File: README.md [hunk 1]
```diff
@@ -42,6 +43,7 @@ Notes about the project structure:
 * The disk_space must be one of the preset choices from the Saturn Cloud UI, it can't be an arbitrary amount of disk space.
 * The startup script must be name `start` without a file extension for Atlas to know it.
 * Options like "environment_variables" may be required even if they are empty, be aware there is a risk in removing them entirely.
+* ``include_in_every_saturn`` indicates whether the example will be created by default in every organizations' Saturn account. Examples that are more niche, or only relevant to certain customers should have ``"include_in_ever_saturn": false``
```
Comment (jameslamb) [inline README.md]:
```suggestion
* ``include_in_every_saturn`` indicates whether the example will be created by default in every organizations' Saturn account. Examples that are more niche, or only relevant to certain customers should have ``"include_in_every_saturn": false``
```

**Review by jameslamb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jsignell) [reply README.md]: gosh dang it!

File: README.md [hunk 2]
```diff
@@ -42,6 +43,7 @@ Notes about the project structure:
 * The disk_space must be one of the preset choices from the Saturn Cloud UI, it can't be an arbitrary amount of disk space.
 * The startup script must be name `start` without a file extension for Atlas to know it.
 * Options like "environment_variables" may be required even if they are empty, be aware there is a risk in removing them entirely.
+* ``include_in_every_saturn`` indicates whether the example will be created by default in every organizations' Saturn account. Examples that are more niche, or only relevant to certain customers should have ``"include_in_every_saturn": false``
```
[the remark this answers is not in the document]
Comment (jsignell) [reply README.md]: OMG...

[the remark this answers is not in the document]
Comment (jsignell) [reply README.md]: My middle school teachers would be ashamed.

**Pull request merged**